UIKitTemplate/UIKitTemplate/AchorViewController.swift

```diff
@@ -0,0 +1,95 @@
+// AchorViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с якорями
```

Comment:
```suggestion
/// Экран реализации поворота светофора с использованием якорей
```
рекомендация на будущее, не обязательно к исправлению